pandas/tests/groupby/test_aggregate.py

```diff
@@ -813,8 +813,6 @@ def test__cython_agg_general(self):
         ('mean', np.mean),
         ('median', lambda x: np.median(x) if len(x) > 0 else np.nan),
         ('var', lambda x: np.var(x, ddof=1)),
-        ('add', lambda x: np.sum(x) if len(x) > 0 else np.nan),
```

Comment:
I moved these to their own tests in https://github.com/pandas-dev/pandas/pull/18921/commits/e390d1cc57bd8f9647ce4b9a0436e7fdd6442a62#diff-1c9535bf6ad763b307a780e37cc4185cR833 due to https://github.com/pandas-dev/pandas/issues/18869